index.js

```diff
@@ -9,8 +10,20 @@ const keyRegex = (str) => {
   return new RegExp('^' + str.replace('*','.*'))
 }
 
+let observer;
+
+const observable = new Observable((_observer) => {
+  observer = _observer;
+  return () => {}
+});
```

Comment:
The problem with the previous implementation was that it only supported one subscription at the time since the `_observer`/`handler` was overwritten on each subscription so only the last one would be called. I just updated the branch with changes that should address this.